OrbitVulkanLayer/SubmissionTrackerTest.cpp

```diff
@@ -660,6 +660,67 @@ TEST_F(SubmissionTrackerTest, StartCaptureDuringSubmissionWontWriteData) {
   tracker_.CompleteSubmits(device_);
 }
 
+TEST_F(SubmissionTrackerTest, MultipleSubmissionsWontBeOutOfOrder) {
+  ExpectFourNextReadyQuerySlotCalls();
+  EXPECT_CALL(dispatch_table_, GetQueryPoolResults)
+      .WillOnce(Return(mock_get_query_pool_results_function_not_ready_))
+      .WillRepeatedly(Return(mock_get_query_pool_results_function_all_ready_));
+
+  std::vector<uint32_t> actual_reset_slots;
+  auto mock_reset_query_slots = [&actual_reset_slots](VkDevice /*device*/,
```

Comment:
Maybe these lambdas are fakes rather than mocks?

Reply:
I guess here both would be fine (as they are also used to validate the result data). But I renamed them.